src/graph/sampling/neighbor/neighbor.cc

```diff
@@ -17,12 +17,57 @@ using namespace dgl::aten;
 namespace dgl {
 namespace sampling {
 
+HeteroSubgraph ExcludeCertainEdges(
```

Comment:
how is this different from calling remove_edges in python?

Reply:
actually, i'm kind of surprised that this solution has speedup over the existing solution.